### DISCUSSION TITLE
postgres:destroy Permission denied

### DISCUSSION THREAD

**Discussion by czepesch** [dokku/dokku#5156] [Q&A]
Hello,
I'm experimenting with deploying some app and created\deleted database few times with the same name.

Now I have error Permission denied running postgres:destroy command.

```
deploy@ubuntu-2gb-fsn1-1:~$ dokku postgres:destroy chuvi_db
 !     WARNING: Potentially Destructive Action
 !     This command will destroy chuvi_db Postgres service.
 !     To proceed, type "chuvi_db"

> chuvi_db
=====> Deleting chuvi_db
/var/lib/dokku/plugins/enabled/postgres/common-functions: line 828: /usr/bin/docker: Permission denied
 !     Service is already stopped
/var/lib/dokku/plugins/enabled/postgres/common-functions: line 379: /usr/bin/docker: Permission denied
       Removing data
/var/lib/dokku/plugins/enabled/postgres/subcommands/destroy: line 49: /usr/bin/docker: Permission denied
```

Please point me to the right direction to fix this.

**Comment by josegonzalez** [comment]
This kind of sounds like the user you are running things as cannot properly change to the dokku user _or_ you somehow changed the permissions on the `dokku` user.

What is the output of `sudo groups dokku`?
